xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -258,31 +259,28 @@ final class DatabaseService {
     }
     
     /// List recent entries
-    func listRecentMemos() -> [EntryStub] {
+    func listRecentMemos() throws -> [EntryStub] {
```

Comment:
Allow listRecentMemos to throw. Previously we were swallowing errors. However while working on this PR I had to troubleshoot this method and found myself wishing we threw. Allowing it to throw revealed the issue lay elsewhere.